transformation/transformationPlugins/jsonMessageTransformers/jsonTypeMappingsSanitizationTransformer/src/test/java/org/opensearch/migrations/transform/TypeMappingsSanitizationTransformerBulkTest.java

```diff
@@ -111,15 +112,31 @@ public void testBulkRequest() throws Exception {
                 "}";
 
 
-        Object resultObj = indexTypeMappingRewriter.transformJson(OBJECT_MAPPER.readValue(testString, LinkedHashMap.class));
-        log.atInfo().setMessage("resultStr = {}").addArgument(() -> {
-            try {
-                return OBJECT_MAPPER.writeValueAsString(resultObj);
-            } catch (JsonProcessingException e) {
-                throw new RuntimeException(e);
+        long totalNanos = 0;
+        final int totalRuns = 10000;
+        int totalCounted = 0;
+        final double warmupPercent = .5;
+        for (int i=0; i<(int)(totalRuns*(1.0+warmupPercent)); ++i) {
+            var input = OBJECT_MAPPER.readValue(testString, LinkedHashMap.class);
+            final var start = System.nanoTime();
+            var resultObj = indexTypeMappingRewriter.transformJson(input);
+            if (i >= totalRuns*warmupPercent) {
+                totalNanos += System.nanoTime() - start;
+                totalCounted++;
             }
-        }).log();
-        Assertions.assertEquals(JsonNormalizer.fromString(expectedString), JsonNormalizer.fromObject(resultObj));
+            log.atDebug().setMessage("resultStr = {}").addArgument(() -> {
+                try {
+                    return OBJECT_MAPPER.writeValueAsString(resultObj);
+                } catch (JsonProcessingException e) {
+                    throw new RuntimeException(e);
+                }
+            }).log();
+            Assertions.assertEquals(JsonNormalizer.fromString(expectedString), JsonNormalizer.fromObject(resultObj));
+        }
+        log.atInfo().setMessage("Total time={} over {} runs")
```

Comment:
Another good case for using JMH, see the first comment for details.

Reply:
Can you provide a link for how to do that?